apps/io/feeding_service_webhook.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2016 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from superdesk.io.registry import registered_webhooks
+import superdesk
+
+
+class FeedingServiceWebhookResource(Resource):
+    public_methods = ['GET', 'POST']
```

Comment:
@petrjasek URL must be generic and is called by third party services, we can't use authentication there because we have no control on the entity doing the HTTP request. In the case of Wufoo, they have a handshake parameter which we could use, but this would be vendor specific.

Furthermore, the risk is limited, as the attacker need to know an URL, and with it he can only trigger a ingestion, there are probably more efficient ways to attack a superdesk instance.

We can add protections like limiting the number of webhook triggers (no more than 1 every X minutes for instance), or restrict to a specific request IP.

This P.R. only add a simple way to trigger ingestion with webhook, but if the security is a concern here, we should discuss it more deeply to decide how to implement the whole thing.

The POST doesn't actually store anything, there is no backend attached to the service.